servers/Azure.Mcp.Server/README.md

```diff
@@ -1,52 +1,208 @@
-# <img height="36" width="36" src="https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/acom_social_icon_azure" alt="Microsoft Azure Logo" /> Azure MCP Server
+<!--
+This readme is used to generate output readme for different package repositories.
+Minor changes to the markdown is required for each package repository as such comment annotations are used to mark parts of the readme that will be updated based on the target package type. Use the eng\scripts\Process-PackageReadMe.ps1 to process the readme
 
-The Azure MCP Server implements the [MCP specification](https://modelcontextprotocol.io) to create a seamless connection between AI agents and Azure services.  Azure MCP Server can be used alone or with the [GitHub Copilot for Azure extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-github-copilot) in VS Code.  This project is in Public Preview and implementation may significantly change prior to our General Availability.
+e.g. eng\scripts\Process-PackageReadMe.ps1 -InputReadMePath "this readme path" -OutputDirectory "output readme directory" -PackageType "nuget, npm, or vsix"
+
+Here are the allowed comment annotations
+
+remove-section: start and remove-section: end is used to remove various lines of markdown for a specified package type
+e.g.
+remove-section: start nuget;vsix;npm
+various markdown lines
+.....
+remove-section: end
+
+remove-chunk: start and remove-chunk: end is used to remove part of the text in a line for a specific package type. Should be used only once on a line
+e.g.
+remove-chunk: start nuget;vsix chunk of text ... remove-chunk: end
+
+insert-chunk is used to insert a chunk of text for a specified package type. Use this only once on a line.
```

Comment:
Is it worth having one vs multiple line options? Perhaps you only need an insert-section and remove-section and support a single line and multi-line syntax.